docs/types.rst

```diff
@@ -914,6 +921,54 @@ Of course, you can also directly access the members of the struct without
 assigning it to a local variable, as in
 ``campaigns[campaignID].amount = 0``.
 
+
+Types as input and output parameters
```

Comment:
"of"?

Reply:
Perhaps we should rather talk about function parameters and return types instead of input and output?

Reply:
@chriseth I think this is a good idea actually, but, the docs use the terms input and output elsewhere, so for consistency, we keep them as they are (and clarify more), or create a new PR that changes them everywhere. Thoughts?